### ISSUE TITLE
[Logging v2] - Chart version 3.9.000 - seccomp.security.alpha.kubernetes.io/defaultProfileName is 

### ISSUE THREAD

**Issue by m4r1u2** [rancher/rancher#31601]
**What kind of request is this:**
bug

**Steps to reproduce:**
- Install Rancher 2.5.6
- Deploy RKE cluster with PSP enabled (restricted) and without any
- Navigate to dashboard and install Logging chart version 3.9.000
- Set ```rbac.psp.enabled = true```

**Result:**
```
-----------------------------------------------------------------------
helm upgrade --install=true --namespace=cattle-logging-system --timeout=10m0s --values=/home/shell/helm/values-rancher-logging-3.9.000.yaml --version=3.9.000 --wait=true rancher-logging /home/shell/helm/rancher-logging-3.9.000.tgz
Release "rancher-logging" does not exist. Installing it now.
creating 16 resource(s)
Error: PodSecurityPolicy.extensions "psp.logging-operator" is invalid: metadata.annotations[seccomp.security.alpha.kubernetes.io/defaultProfileName]: Invalid value: "docker/default,runtime/default": must be a valid seccomp profile
-----------------------------------------------------------------------
```

Is it correct syntax in [psp.yaml#L8](https://github.com/rancher/charts/blob/release-v2.5/charts/rancher-logging/rancher-logging/3.9.000/templates/psp.yaml#L8) ?

If deployed with ```rbac.psp.enabled = false``` is fine, but then without ```psp:psp.logging-operator``` containg the seccomp defaultProfileName.

## More debugging:

**This PSP is ok:**
```
apiVersion: policy/v1beta1
kind: PodSecurityPolicy
metadata:
  annotations:
    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'runtime/default'
    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
```
**Also OK:**
```
apiVersion: policy/v1beta1
kind: PodSecurityPolicy
metadata:
  annotations:
    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
```
**Not OK:**
```
apiVersion: policy/v1beta1
kind: PodSecurityPolicy
metadata:
  annotations:
    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default,runtime/default'
    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
```

**Environment information**
- Rancher version: v2.5.6
- Installation option: HA RKE

Reference: rancher/rancher#30771 rancher/rancher#31006

**Comment by izaac** [comment]
@m4r1u2 I reproduced this with the following steps:

- Install Rancher version 2.5.6
- Provision an RKE Downstream cluster
- Install the logging v2 chart with default values from Dashboard.  #30771
- Then Edit/Upgrade the Installed chart 
  - Go to `Installed Apps`, edit the `rancher-logging`
  - Go to `Chart Options` -> `Edit as YAML`

Change this block from:

```yaml
rbac:
  enabled: true
  psp:
    enabled: false
```

To:

```yaml
rbac:
  enabled: true
  psp:
    enabled: true
```
- Click Upgrade.

`Error: UPGRADE FAILED: failed to create resource: PodSecurityPolicy.extensions "psp.logging-operator" is invalid: metadata.annotations[seccomp.security.alpha.kubernetes.io/defaultProfileName]: Invalid value: "docker/default,runtime/default": must be a valid seccomp profile `

**Comment by Martin-Weiss** [comment]
Just ran into this with rancher 2.5.7 against RKE2 downstream cluster with cis-1.5 - any idea when we can expect a fix?

**Comment by bashofmann** [comment]
I think this was fixed upstream with https://github.com/banzaicloud/logging-operator/pull/686. This PR is included the upstream logging chart 3.9.1 which should be included with Rancher 2.5.8.

**Comment by izaac** [comment]
Assigning this old issue to Charts QA SME for follow up depending on priorities.
